sophus/common.hpp

```diff
@@ -39,6 +39,8 @@
 #define FMT_STRING_ALIAS 1
 #endif
 
+#define FMT_HEADER_ONLY
```

Comment:
Possibly this should be defined at the cmake level.  I worry this might mess with other users of the library.

Reply:
See: https://fmt.dev/latest/usage.html#usage-with-cmake

Reply:
yes, using the fmt::fmt-header-only target instead